pkg/doc/vc/crypto/crypto.go

```diff
@@ -27,15 +28,20 @@ type keyResolver interface {
 }
 
 type signer struct {
-	kms   legacykms.KMS
-	keyID string
+	kms        legacykms.KMS
```

Comment:
maybe it makes sense to separate signers?

Reply:
done